documentation/docs/04-hooks.md

```diff
@@ -73,9 +73,9 @@ export function getSession({ context }) {
 
 ### handle
 
-This function runs on every request, and determines the response. The second argument, `render`, calls SvelteKit's default renderer. This allows you to modify response headers or bodies, or bypass SvelteKit entirely (for implementing endpoints programmatically, for example).
+This function runs on every request, and determines the response. It also receives `render`, which calls SvelteKit's default renderer. This allows you to modify response headers or bodies, or bypass SvelteKit entirely (for implementing endpoints programmatically, for example).
```

Comment:
It sounds a bit off to me to say "also receives" without mentioning anything else that is received

```suggestion
This function runs on every request, and determines the response. It receives the `request` and `render`, which calls SvelteKit's default renderer. This allows you to modify response headers or bodies, or bypass SvelteKit entirely (for implementing endpoints programmatically, for example).
```

Reply:
yeah, this reads better to me